src/main/java/StringCalculator/view/CalculatorView.java

```diff
@@ -0,0 +1,21 @@
+package StringCalculator.view;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Scanner;
+import java.util.regex.Pattern;
+
+public class CalculatorView {
+
+    private final static Pattern SPACE = Pattern.compile(" ");
+    private static final Scanner scanner = new Scanner(System.in);
```

Comment:
상수 컨벤션에 따라 이름은 대문자로 작성해보면 어떨까요?
```suggestion
    private static final Scanner SCANNER = new Scanner(System.in);
```